### PR TITLE
Add ngff rename script as a management command

### PR DESCRIPTION
**Pull request by mvandenburgh**
Adds the script I wrote to rename .ngff files to .ome.zarr as a management command so we have it for future reference

### CODE REVIEW

**Review by AlmightyYakob — DISMISSED** [review]
Looks good overall, just one suggestion

File: dandiapi/api/management/commands/rename_ngff.py
Comment (mvandenburgh) [reply dandiapi/api/management/commands/rename_ngff.py]: Good catch, I restricted it to the current zarr's dandiset https://github.com/dandi/dandi-archive/pull/1120/commits/38c7dd1e1b0aab4b046491c3237aee087f1f3c75. `ZarrArchives` are associated with `Dandisets` and not `Versions`, so I believe that is sufficient

**Comment by mvandenburgh** [comment]
@AlmightyYakob can you review the changes here?

File: dandiapi/api/management/commands/rename_ngff.py
Comment (AlmightyYakob) [inline dandiapi/api/management/commands/rename_ngff.py:37]: Should we use `.iterator` here?

Comment (mvandenburgh) [reply dandiapi/api/management/commands/rename_ngff.py:37]: I suppose so, since a `ZarrArchive` could have a lot of associated assets, and the primary use case for this script would be running it in a `heroku run` environment (where we're constrained by the dyno's memory quota).

**Comment by yarikoptic** [comment]
FWIW dandi move command could be used. See https://github.com/dandi/dandi-cli/issues/1081#issuecomment-1191798525 for an example with regex.

**Comment by mvandenburgh** [comment]
> FWIW dandi move command could be used. See [dandi/dandi-cli#1081 (comment)](https://github.com/dandi/dandi-cli/issues/1081#issuecomment-1191798525) for an example with regex.

Oh that might be a better option for this kind of operation than a management command. @satra requested that this script be committed to the repo, so I'll leave it up to him whether or not this should still be merged.

**Review by AlmightyYakob — APPROVED** [review]
LGTM if we end up wanting to merge.

**Comment by yarikoptic** [comment]
> Jun 15

was the date for this PR birth... `dandi move` https://github.com/dandi/dandi-cli/pull/1024 was initiated May 31st but merged into master of dandi-cli Jun 27 .  I do feel that this PR is just no longer needed, but @satra might have different thoughts.

**Comment by satra** [comment]
i think this is a more generic operation at the level of the database if necessary. i would keep it in the folder as an example of how to operate/rename on many assets, but most likely we will never use this specific script (there should be no ngff files moving forward).

**Pull request merged**
